cli/cmd/cmdutil/cmdutil.go

```diff
@@ -135,6 +138,32 @@ func InputPrompt(msg, def string) string {
 	return result
 }
 
+func ProjectExists(ctx context.Context, c *client.Client, orgName, projectName string) (bool, error) {
+	resp, err := c.GetProject(ctx, &adminv1.GetProjectRequest{OrganizationName: orgName, Name: projectName})
+	if err != nil {
+		if st, ok := status.FromError(err); ok {
+			if st.Code() == codes.NotFound {
+				return false, nil
+			}
+		}
+		return false, err
+	}
+	return resp.Project.Name == projectName, nil
```

Comment:
Actually, this should just return `true` (names are case insensitive, but `==` is a case sensitive comparison)